src/main/java/org/broadinstitute/hellbender/tools/walkers/vqsr/CNNScoreVariants.java

```diff
@@ -285,6 +279,61 @@ public void apply(final VariantContext variant, final ReadsContext readsContext,
         sendBatchIfReady();
     }
 
+    @Override
+    public void afterFirstPass() {
+        if (waitforBatchCompletion) {
+            pythonExecutor.waitForPreviousBatchCompletion();
+        }
+        if (curBatchSize > 0) {
+            executePythonCommand();
+            pythonExecutor.waitForPreviousBatchCompletion();
+        }
+
+        pythonExecutor.sendSynchronousCommand("tempFile.close()" + NL);
+        pythonExecutor.terminate();
+
+        try {
+            scoreScan = new Scanner(scoreFile);
+            vcfWriter = createVCFWriter(new File(outputFile));
+            scoreScan.useDelimiter("\\n");
+            writeVCFHeader(vcfWriter);
+        } catch (IOException e) {
+            throw new GATKException("Error when trying to write annotated VCF.", e);
```

Comment:
It could be the score file - might be worth having a nested try/catch for that.

Reply:
Also, the Scanner is never closed anywhere.

Reply:
it seems createVCFWriter wont throw IOExceptions, so I just fixed the error message. Scanner is now closed.